scripts/helper_scripts/unipept-database-rs/src/bin/xml-parser.rs

```diff
@@ -4,6 +4,7 @@ use std::num::NonZeroUsize;
 use anyhow::{Context, Result};
 use clap::Parser;
 use smartstring::{LazyCompact, SmartString};
+use unipept_database::uniprot::UniprotType;
```

Comment:
This thing checks whether the Uniprot type is set to either `swissprot` or `trembl`, but these are not the two only options. We can also use this script to process and download so-called "reference proteomes" that can have any ID (mostly in the format U000xxxx, but that's not always the case). So I think we should remove this check here.

Reply:
Didn't know this was possible ^^ I agree we should remove it then